code/_onclick/adjacent.dm

```diff
@@ -102,14 +102,14 @@ Quick adjacency (to turf):
 	This can be safely removed if border firedoors are ever moved to be on top of doors
 	so they can be interacted with without opening the door.
 */
-/obj/machinery/door/Adjacent(var/atom/neighbor)
+/*/obj/machinery/door/Adjacent(var/atom/neighbor)
```

Comment:
It is exclusively for a typed of firedoor called border_only that I have also commented out. Border_only inherited from firedoors entirely because all of its relevant code was commented out by some predecessor developer for being "too expensive".